shared/aux.go

```diff
@@ -45,7 +46,7 @@ func RunCommandOnNode(cmd, ip string) (string, error) {
 	if cmd == "" {
 		return "", ReturnLogError("cmd should not be empty")
 	}
-
```

Comment:
add a new empty line here please and after the log statement

Reply:
added 👍